ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml

```
@@ -0,0 +1,23 @@
+binary_build_commands: make manager
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: unit
+  commands: ci/prow/tests-unit
```

```suggestion
  commands: ci/prow/unit_tests[.sh]
```
Can we try to be consistent with the makefile targets and the scripts names?